src/CardUtils.res

```diff
@@ -671,3 +671,11 @@ let getEligibleCoBadgedCardSchemes = (~matchedCardSchemes, ~enabledCardSchemes)
     enabledCardSchemes->Array.includes(ele->String.toLowerCase)
   })
 }
+
+let getCardBrandFromStates = (cardBrand, cardScheme, showFields, isNotBancontact) => {
+  switch (showFields, isNotBancontact) {
+  | (false, _) => cardScheme
+  | (_, true) => cardBrand
+  | _ => cardBrand
+  }
+}
```

Comment:
```suggestion
let getCardBrandFromStates = (cardBrand, cardScheme, showFields, isNotBancontact) =>
  !showFields ? cardScheme : cardBrand;
```

Reply:
Sure @PritishBudhiraja, logically this seems more straightforward. @preetamrevankar could be please test CVC fields in card form, saved cards screen and **Bancontact** card payment method?